lib/chains/llm_chain.ex

```diff
@@ -307,6 +321,7 @@ defmodule LangChain.Chains.LLMChain do
     deltas
     |> List.flatten()
     |> Enum.reduce(chain, fn d, acc -> apply_delta(acc, d) end)
+    |> delta_to_message()
```

Comment:
This is fixing a bug I noticed where if the stream only returns a single `MessageDelta` (this happens for streamed function calls for GoogleAI), the tracking delta is never actually converted to a `Message` and the chain will enter an infinite loop.